docs/developer/creating-a-plugin.rst

```diff
@@ -42,13 +67,16 @@ This means that it will be a folder named **scipion-em-package** with the struct
 (explained below in the second part) and it will contain a folder **package** that is a Python module
 with functions and classes required by Scipion (explained below).
 
-Plugin Overview
-===============
+Code migration from Scipion to an independent plugin
```

Comment:
@JorMaister , I think this can be removed now. All it's being migrated.

Reply:
It has been currently removed.

Thanks.